### PR TITLE
perf(core/clusters): use ReactVirtualized to render clusters

### PR DESCRIPTION
**Pull request by anotherchrisberry**
Now that the clusters view is almost entirely React, we can take advantage of React Virtualized to render the view.

For sufficiently large apps (dozens of clusters, thousands of instances), we are seeing a pretty significant performance improvements on the rendering of the view, cutting between 1 and 2 seconds off the render. When the "with details" box is checked, the improvement is significantly more noticeable.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
   },
   "dependencies": {
     "@spinnaker/kayenta": "0.0.19",
+    "@types/react-virtualized": "9.7.2",
```
Comment (jrsquared) [inline package.json]: dev dependency

Comment (anotherchrisberry) [reply package.json]: yes

**Comment by jrsquared** [comment]
v.nice

File: app/scripts/modules/core/src/cluster/AllClustersGroupings.tsx [hunk 2]
```diff
@@ -13,71 +18,106 @@ export interface IAllClustersGroupingsProps {
...
+    const parent: React.ReactType = props.parent as any; // bad cast, sad @types
+    return (
+      <CellMeasurer
+        cache={this.cellCache}
+        key={props.key}
+        rowIndex={props.index}
+        parent={parent}
+      >
+        <div key={props.index} style={props.style}>
+          <div>
```
Comment (archana-s) [inline app/scripts/modules/core/src/cluster/AllClustersGroupings.tsx]: this <div> is not necessary

Comment (anotherchrisberry) [reply app/scripts/modules/core/src/cluster/AllClustersGroupings.tsx]: Whoops, you are right. This was an artifact of debugging issues with the CellMeasurer. Will remove in a subsequent PR

**Pull request merged**